editor/src/clj/editor/editor_extensions/ui_components.clj

```diff
@@ -494,6 +494,7 @@
     {:fx/type ext-with-extra-check-box-props
      :desc (-> {:fx/type fx.check-box/lifecycle
                 :style-class ["check-box" "ext-check-box"]
+                :mnemonic-parsing false
```

Comment:
Somewhere it bit me... maybe in a check-box that had a label like `x86_64-linux`